react/features/base/conference/actions.js

```diff
@@ -222,8 +222,7 @@ export function conferenceLeft(conference: Object) {
 function _conferenceWillJoin(conference: Object) {
     return (dispatch: Dispatch<*>, getState: Function) => {
         const localTracks
-            = getState()['features/base/tracks']
-                .filter(t => t.local)
+            = getLocalTracks(getState()['features/base/tracks'])
```

Comment:
This function has a confusing name because it returns jitsi tracks, not tracks.

Reply:
No, it does not. It returns a Track not JitsiTrack

Reply:
Ah, sorry, misread it!